### DISCUSSION TITLE
[Feature] View search results in context

### DISCUSSION THREAD

**Discussion by volcs0** [immich-app/immich#8514] [Feature Request]
### The feature

Often I want to find a photo but I can only remember something about the other photos in the group. For example, I might want a photo of my kids in the playground at Niagara Falls. I search Niagara falls and find some pictures from that trip. Now I want to go to that day and see the other pictures from the same time period. Google Photos has had this feature for a long time - you search for a photo and then you can see all the photos from that same day. This would be a very useful feature, I think. Thanks for your consideration.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by mertalev** [comment]
A "jump to timeline" feature is planned. But due to the complexity of how the timeline is built, it likely won't be done in the short term.

**Comment by bo0tzz** [comment]
Duplicate of #4115
